web/src/settings_org.js

```diff
@@ -1109,6 +1175,11 @@ export function build_page() {
         update_message_edit_sub_settings(is_checked);
     });
 
+    $("#id_realm_allow_resolve_topic").on("change", (e) => {
+        const is_checked = $(e.target).prop("checked");
+        update_resolve_topic_sub_settings(is_checked);
+    });
+
```

Comment:
This is for disabling/enabling the dropdown depending on the `allow_resolve_topic` checkbox.